src/components/dialog/CreateIssueTemplateDialog.vue

```diff
@@ -0,0 +1,852 @@
+                          label="Inverse Name"
+                          class="mx-2 mb-2"
+                          density="compact"
+                          :error="!!inverseNameErrorMessage"
+                          :error-messages="inverseNameErrorMessage"
+                        />
+                      </template>
+                  </ExpandableCard>
+              </v-col>
+            </v-row>
+          </v-form>
+        </template>
+
+        <!-- Step 4: Issue States -->
+        <template v-slot:item.4>
+          <v-form v-model="formIssueStatesValid">
+            <v-combobox v-model="issueStates" label="Issue States" multiple chips clearable class="mb-2" />
+          </v-form>
+        </template>
+
+        <!-- Step 5 - 7: Placeholder Steps -->
+        <template v-slot:item.5>
+          <v-form v-model="formAssignmentTypeValid">
+            <v-alert type="info">Assignment Type (Placeholder)</v-alert>
+          </v-form>
+        </template>
+        <template v-slot:item.6>
+          <v-form v-model="formRelationTypeValid">
+            <v-alert type="info">Relation Type (Placeholder)</v-alert>
+          </v-form>
+        </template>
+        <template v-slot:item.7>
+          <v-form v-model="formVersionTemplatesValid">
+            <v-alert type="info">Component Version Templates (Placeholder)</v-alert>
+          </v-form>
+        </template>
+        <template v-slot:item.8>
+          <v-form v-model="formTemplateFieldsValid">
+            <v-alert type="info">Template Field Specifications (Placeholder)</v-alert>
+          </v-form>
+        </template>
+
+      </v-stepper>
+
+      <v-card-actions>
+        <DefaultButton variant="text" :disabled="step === 1" @click="previous">Previous</DefaultButton>
+        <v-spacer />
+        <DefaultButton variant="text" color="" @click="!isDirty && cancelCreateIssueTemplate()">
+                Cancel
+                <ConfirmationDialog
+                    v-if="isDirty"
+                    :title="`Discard Issue Template?`"
+                    :message="`Are you sure you want to discard this issue template?`"
+                    confirm-text="Discard"
+                    @confirm="cancelCreateIssueTemplate"
+                />
+        </DefaultButton>
+        <DefaultButton variant="text" color="primary" @click="next">{{ step === stepLabels.length ? 'Create' : 'Next' }}</DefaultButton>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue';
+import { useForm, defineField } from 'vee-validate';
+import * as yup from 'yup';
+import { fieldConfig } from '@/util/vuetifyFormConfig';
+import { onEvent } from '@/util/eventBus';
+import { computed } from "vue";
+import ConfirmationDialog from "./ConfirmationDialog.vue";
+import ExpandableCard from '../ExpandableCard.vue';
+
+
+import { iconList as baseIconList } from '../../assets/icons';
```

Comment:
imho just using all the icons provided by some library would probably be the best choice here
I think lucide would be good (but you would need to filter for icons which can be converted to a single path, but should be pretty easy) (conversions would need to be done for line, polyline, ellipse, rect, circle)

Reply:
@mdi/js would also be an option, there the icons are already provided as path